examples/nlp/language_modeling/conf/config.yaml

```diff
@@ -0,0 +1,106 @@
+# BERT Pretraining
+trainer:
+  gpus: 1 # the number of gpus, 0 for CPU, or list with gpu indices
+  num_nodes: 1
+  max_epochs: 2 # the number of training epochs
+  max_steps: null # precedence over max_epochs
+  accumulate_grad_batches: 1 # accumulates grads every k batches
+  precision: 16 # 16 to use AMP 
+  amp_level: O1 # O1 or O2 if using AMP 
+  distributed_backend: ddp
+  gradient_clip_val: 0.0
+  row_log_interval: 1
+  val_check_interval: 1.0 # check once per epoch .25 for 4 times per epoch
+  checkpoint_callback: false # provided by exp_manager
+  logger: false # provided by exp_manager
+
+model:
+  pretrained_model_name: bert-base-uncased # huggingface model name
+  max_seq_length: 128
+  # The maximum total input sequence length after tokenization. Sequences longer than this
+  # will be truncated, and sequences shorter than this will be padded.
+  mask_prob: 0.15
+  # Probability of masking a token in the input text during data processing.
+  short_seq_prob: 0.1
+  # Probability of having a sequence shorter than the maximum sequence length `max_seq_length` in data processing.",
+  batch_size: 16 # per GPU
+
+  language_model:
+    pretrained_model_name: ${model.pretrained_model_name}
```

Comment:
why are you duplication this?

Reply:
This is interpolation. The user only has to input once in the config and then then input can be used for other configurations.